packages/vue/src/components/organisms/SfMegaMenu/SfMegaMenu.stories.scss

```diff
@@ -20,4 +20,10 @@
       }
     }
   }
+  &__sf-bar {
```

Comment:
I think this isn't the right file for this change. What do you think about relocating this part to `SfMegaMenu`? :)